introduction.md

```diff
@@ -1 +1,6 @@
 Hi, I'm JR. My favorite path is Javascript/Angular.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?